lib/chef/cookbook/synchronizer.rb

```diff
@@ -131,7 +131,7 @@ def mark_file_synced(file)
       files_remaining_by_cookbook[file.cookbook] -= 1
 
       if files_remaining_by_cookbook[file.cookbook] == 0
-        @events.synchronized_cookbook(file.cookbook.name)
+        @events.synchronized_cookbook(file.cookbook.name, file.cookbook.version)
```

Comment:
Are we concerned about breaking 3rd party formatters here?  If they implemented the old single-argument method, they will get an ArgumentError here.